frontend/www/js/omegaup/components/arena/FeedbackCodeView.vue

```diff
@@ -106,12 +118,34 @@ export default class FeedbackCodeView extends Vue {
     };
   }
 
+  setFeedback({
+    lineNumber,
+    text = null,
+    status,
+  }: {
+    lineNumber: number;
+    text: string | null;
+    status: FeedbackStatus;
+  }): void {
+    this.feedbackMap.set(lineNumber, {
+      lineNumber,
+      text,
+      status,
+    });
+    this.mapChangeTracker++;
+  }
+
+  deleteFeedback({ lineNumber }: { lineNumber: number }): void {
+    this.feedbackMap.delete(lineNumber);
+    this.mapChangeTracker--;
```

Comment:
```suggestion
    this.mapChangeTracker++;
```

para que el truco funcione, debemos garantizar que el tracker jamás va a ser negativo. y en realidad lo único que importa es que cambie de valor, no _necesita_ ser igual al número de elementos (que igual ni si quiera se está garantizando, porque se puede mandar llamar `setFeedback` múltiples veces con la misma línea).

Reply:
Listo